contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortBuildAndPushJob.java

```diff
@@ -309,6 +312,8 @@ public VoldemortBuildAndPushJob(String name, azkaban.utils.Props azkabanProps) {
         }
         this.executorService = Executors.newFixedThreadPool(requiredNumberOfThreads);
 
+        this.storeVerificationExecutorService = Executors.newFixedThreadPool(THREAD_NUM_FOR_STORE_VERIFICATION);
```

Comment:
Actually, I want to reuse the thread pool, which can be used by multiple cluster verification.
Otherwise, I need to create a new pool for each cluster verification.

Since there is no requirement to add more pre-process steps right now, the current logic should be acceptable since it is just store verification.
If there are more steps later on, we can wrap them up to a separate function.